test/strategy/auth0_test.exs

```diff
@@ -47,413 +53,387 @@ defmodule Ueberauth.Strategy.Auth0Test do
   end
 
   # Tests:
-  test "simple oauth2 /authorize request" do
-    conn =
-      :get
-      |> conn("/auth/auth0")
-      |> SpecRouter.call(@router)
-
-    assert conn.resp_body =~ ~s|<html><body>You are being <a href=|
-    assert conn.resp_body =~ ~s|>redirected</a>.</body></html>|
-    assert conn.resp_body =~ ~s|href="https://example-app.auth0.com/authorize?|
-    assert conn.resp_body =~ ~s|client_id=clientidsomethingrandom|
-    assert conn.resp_body =~ ~s|redirect_uri=http%3A%2F%2Fwww.example.com%2Fauth%2Fauth0%2Fcallback|
-    assert conn.resp_body =~ ~s|response_type=code|
-    assert conn.resp_body =~ ~s|scope=openid+profile+email|
-    assert conn.resp_body =~ ~s|state=#{conn.private[:ueberauth_state_param]}|
-  end
+  describe "handle_request!" do
```

Comment:
Could you split the test refactoring into 2 separate commits or, even better, into two separate pull requests ? This will make the reviewing easier by first moving the tests into `describe` blocks with no changes and *then* add tests for the current PR. Right now I'm having some trouble to find what has really changed from what is a simple refactor ? Thanks :)

Reply:
I've actually refactored the existing tests which are on the `master` branch as well as fixed the CI tests. If you rebase it should be quite fast to update your branch :)